system/logs/templates/_openstack-config.tpl

```diff
@@ -0,0 +1,214 @@
+{{- define "openstack.receiver" }}
+filelog/containerd:
```

Comment:
Similar to above but defined here https://github.com/sapcc/helm-charts/blob/8bc802b3c899c31ac07836459715f85c502a5322/system/logs/templates/_containerd-config.tpl#L24